pokemon-service/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/service/TrainerService.java

```diff
@@ -101,4 +85,12 @@
      * of old password failed
      */
     public boolean changePassword(Trainer trainer, String oldPassword, String newPassword);
+    
+    /**
+     * Finds all pokemons of a trainer.
+     * 
+     * @param trainer The trainer to find pokemons of.
+     * @return List of all pokemons of the trainer.
+     */
+    public List<Pokemon> getOwnPokemons(Trainer trainer);
```

Comment:
Plural of Pokemon is Pokemon (same as with 1 species - 2 species), also surely this should be in PokemonService (like it is in DAO and Facade layer). Then it can also have consistent naming - getPokemonOfTrainer.